actors/runtime/runtime.go

```diff
@@ -170,3 +172,8 @@ type CBORMarshalable interface {
 type CBORUnmarshalable interface {
 	UnmarshalCBOR(r io.Reader) error
 }
+
+type CBORAble interface {
+	CBORMarshalable
+	CBORUnmarshalable
+}
```

Comment:
noting that here and elsewhere will conflict with https://github.com/filecoin-project/specs-actors/pull/45/files#diff-e2d8254b9ce8e65885afe1cf6a808a1dR173-R177

Reply:
Yeah, sorry @whyrusleeping . I changed the interface names to match the more go-idiomatic ones from cbg.